snapraid-runner.py

```diff
@@ -288,6 +296,17 @@ def run():
             finish(False)
         logging.info("*" * 60)
 
+    if config["smart"]["enabled"]:
+        logging.info("Running SMART...")
+        # Expand PATH to include /usr/sbin for running smartctl from cron
+        os.environ["PATH"] += os.pathsep + "/usr/sbin"
```

Comment:
This is clearly not cross-platform :). Even on unix it makes strong assumptions about the user's setup. I assume you had problems with the binary not being found? Why don't you make the binary path a setting?

Reply:
Good point, I definitely overlooked the fact that snapraid is cross-platform and wrote this for Linux.

Reply:
You should probably remove this and just make sure the command is run in the correct environment.

Reply:
It look very comprehensive, but I don't think this script should make any attempt to find the executable. That is just very complicated and the PATH system of the OSs is perfectly suitable to handle this job. I will have a look and do some changes to this PR.